lib/Driver/Driver.cpp

```diff
@@ -84,7 +84,7 @@ void Driver::parseDriverKind(ArrayRef<const char *> Args) {
   std::string OptName;
   // However, the driver kind may be overridden if the first argument is
   // --driver-mode.
-  if (!Args.empty()) {
+  if (Args.size() > 0) {
```

Comment:
Why this change? `empty()` is idiomatic C++ like it is in Swift because `size()` may be expensive to compute.

Reply:
Thanks for catching it. It was a mistake: I compared the current version to an older one and picked the wrong old version to compare it to.